corehq/motech/repeaters/static/repeaters/js/bootstrap3/repeat_record_report.js

```diff
@@ -367,18 +367,10 @@ hqDefine('repeaters/js/bootstrap3/repeat_record_report', [
             }
         });
 
-        function selectItems() {
+        function toggleItems(checked) {
```

Comment:
I don't think this is an improvement, because it isn't obvious to the reader what the true/false parameter means, you have to go to the function definition to look it up.

Reply:
37834d433c079d41c7d205230f0edba4f99a91a5